yt_dlp/extractor/ard.py

```diff
@@ -636,32 +636,30 @@ def _real_extract(self, url):
                     'Content-Type': 'application/json'
                 })['data']['playerPage']
 
-            content_id = str_or_none(try_get(
-                player_page, lambda x: x['tracking']['atiCustomVars']['contentId']))
+            raw_player_page = player_page
         except ExtractorError as e:
-            # try to get data from website API endpoint as public gateway failed with 503
-            if isinstance(e.cause, HTTPError) and e.cause.status == 503:
+            if not isinstance(e.cause, HTTPError) or e.cause.status != 503:
+                raise
+            else:  # try to get data from website API endpoint as public gateway failed with 503
                 player_page = self._download_json(
-                    f"https://api.ardmediathek.de/page-gateway/pages/ard/item/{video_id}", display_id, headers={
+                    f'https://api.ardmediathek.de/page-gateway/pages/ard/item/{video_id}', display_id, headers={
                         'Content-Type': 'application/json'
-                    }
+                    }, note='Downloading fallback JSON metadata'
                 )
```

Comment:
This for example is a hanging indent. Generally we try to avoid these, and collapse them into the line above, like so:
```suggestion
                    }, note='Downloading fallback JSON metadata')
```